src/jit/gentree.h

```diff
@@ -2671,7 +2671,7 @@ struct GenTreeCall final : public GenTree
 
 #ifdef FEATURE_READYTORUN_COMPILER
     // Call target lookup info for method call from a Ready To Run module
-    CORINFO_CONST_LOOKUP gtEntryPoint;
+    CORINFO_LOOKUP gtEntryPoint;
```

Comment:
Reverted the change

---
In reply to: [61362958](https://github.com/dotnet/coreclr/pull/4519#discussion_r61362958) [](ancestors = 61362958)